conda-recipe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: pmgr
+  version : {{ data.get('version') }}
+
+source:
+  path: ..
+
+build:
+  number: 1
+  script:
+    python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python {{PY_VER}}*,>=3.6
+    - setuptools
+
+  run:
+    - python {{PY_VER}}*,>=3.6
+    - pyqt =5.6
+    - pyca >=3
+    - pykerberos >=1.1.14
```

Comment:
Saw note in commit. This is the same package used by the `elog` so should be fine.